.travis.yml

```diff
@@ -16,24 +22,28 @@ before_script:
   - psql -c 'create extension postgis;' -U postgres
 
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
+#  - 2.0
```

Comment:
remove comments???

Reply:
I'll keep them for now. We can delete those later if we decide not to support these versions.